tests/rodeos_wasmQL_http_timeout.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+from testUtils import Utils
+from TestHelper import TestHelper
+from rodeos_utils import RodeosCluster
+from TestHelper import AppArgs
+import time
+import os
+import logging
+
+try: 
+    import requests
+except ImportError:
+    os.system('pip3 install requests')
+    import requests
+try: 
+    import requests_unixsocket
+except ImportError:
+    os.system('pip3 install requests_unixsocket')
+    import requests_unixsocket
```

Comment:
In the CICD Dockerfile builds you are guaranteeing these two modules are installed. Therefore the above logic should be unnecessary.

Reply:
Thanks, removed.